hack/tools.mk

```
@@ -22,3 +24,6 @@ tool_version_file = $(TOOLS_BIN_DIR)/.version_$(subst $(TOOLS_BIN_DIR)/,,$(1))_$
 
 $(COSIGN): $(call tool_version_file,$(COSIGN),$(COSIGN_VERSION))
 	GOBIN=$(abspath $(TOOLS_BIN_DIR)) go install github.com/sigstore/cosign/v2/cmd/cosign@$(COSIGN_VERSION)
+$(CRANE): $(call tool_version_file,$(CRANE),$(CRANE_VERSION))
```

Just add one empty line to improve readability.

```suggestion

$(CRANE): $(call tool_version_file,$(CRANE),$(CRANE_VERSION))
```